FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Models/Jobs/Job.cs

```diff
@@ -124,6 +134,12 @@ public class Job
         [JsonProperty("failedReason")]
         public string FailedReason { get; set; }
 
+        /// <summary>
+        /// Failure exception for this job.
+        /// </summary>
+        [JsonProperty("failedException")]
+        public Exception FailedException { get; set; }
```

Comment:
Do we have tested this **Exception** can be correctly parsed and serialized between JSON content in blob and runtime **Job** instance?

Reply:
From the codes, seems we only want to know the exception type and save that to notifictaion?